front/components/assistant/conversation/ConversationContainer.tsx

```diff
@@ -231,12 +231,12 @@ export function ConversationContainer({
         });
       } else {
         // We start the push before creating the message to optimize for instantaneity as well.
-        setActiveConversationId(conversationRes.value.sId);
-        void router.push(
+        await router.push(
```

Comment:
required so that we snap once the shallow URL change is done so that everyone snap at the same time (title in, conversation in, assistant out)